org.gabmus.notorious.json

```diff
@@ -0,0 +1,78 @@
+{
+    "app-id": "org.gabmus.notorious",
+    "command": "notorious",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home:rw"
```

Comment:
What does it need `home:rw` for?

Reply:
Users can choose a folder to use for notes in their home directory. This is useful if one wants to use a folder that's synced with an online service like Nextcloud. I personally use `~/Nextcloud/Documents/Notes`.

Besides the default is `~/Documents/Notes` and there's a good reason not to keep these notes inside of the sandbox like moving them around, sharing them or doing stuff like grep on them.

Reply:
I noticed it in the settings after posting my quesiton. :+1: